### ISSUE TITLE
Associate the Figure Caption with a css class

### ISSUE THREAD

**Issue by tinyheero** [adletaw/captioner#23]
Hi,

Do you have a suggested way or method to associate the captions with some sort of css class? It would be nice to have an easy way to have the figure caption appear as a different color or even bolded. For instance:

```
fig_nums <- captioner(prefix = "Figure", css.class = "figcaption")
test.cap <- fig_nums("test", "test caption")
```

When using `r test.cap` would give something like:

```
<p class="figcaption">Figure 1: SNV Mutational Load Distribution</p>
```

Then the figcaption css class could be styled which ever way you want. Even better if there was the granularity to associate the prefix with some class that is separate from the actual caption. This could allow for like the prefix to be italicized while the caption could be like bolded:

*Figure 1*: **SNV Mutational Load Distribution**

Thanks,

Fong

**Comment by adletaw** [comment]
I don't have a way to do this at the moment, but it is an idea I have been thinking about.  I will be getting back to active development in the near future.

**Comment by adletaw** [comment]
Pull request merged (see #24)
